### PR TITLE
(feat) O3-2637: Rename `Test Results` left panel link to `Results Viewer`

### PR DESCRIPTION
**Pull request by jwnasambu**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) label. See existing PR titles for inspiration.
- [ ] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://zeroheight.com/23a080e38/p/880723-introduction).
- [ ] My work includes tests or is validated by existing tests.

## Summary
I renamed the `Test Results ` on the left nav to `Results Viewer `by  changing the `path` property  on the dashboardMeta and` routes.json` file and `title `in the dashboardMeta object to identify the "Results Viewer" 

## Screenshots
Screenshot 2023-12-13 at 6 42 26 PM

## Related Issue
https://openmrs.atlassian.net/browse/O3-2637)

### CODE REVIEW

**Comment by jwnasambu** [comment]
@denniskigen Am sorry I didn't push the changes made in the `package.json` file my assumption was it was meant for me to confirm its working on my side. It worked as expected on the side nav and breadcrumb!

**Comment by denniskigen** [comment]
That's fine, @jwnasambu. The changes to `package.json` ought to be removed once you're done testing your changes. See the last step in https://o3-docs-git-feat-common-lib-dev-mode-denniskigen.vercel.app/docs/frontend-modules/development#running-esm-patient-common-lib-locally. 

Also, the correct conventional commit label for this PR should be `(feat)` because it's changing something in the UI.

**Comment by jwnasambu** [comment]
@denniskigen thanks so much for the review and guidance. I really appreciate you help!

File: packages/esm-patient-labs-app/src/routes.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
       "meta": {
         "slot": "patient-chart-test-results-dashboard-slot",
         "columns": 1,
-        "path": "Test Results",
+        "path": "Results Viewer",
```
Comment (pirupius) [inline packages/esm-patient-labs-app/src/routes.json:15]: 👍🏾

**Review by pirupius — APPROVED** [review]
Thanks @jwnasambu

**Comment by pirupius** [comment]
@denniskigen this was a requirement as part of the ongoing work with labs so am not sure if there any blockers or reservations to this being merged. If not, please go ahead with the merge.

**Comment by denniskigen** [comment]
At the bare minimum, it'd be ideal to provide additional context supporting this change, such as:

- Links to the new designs
- At least a link to the epic and wiki

**Comment by denniskigen** [comment]
AFAICT, this is the only design I've seen with the name change https://zpl.io/noln0Ar

**Pull request merged**